zowe-cli-id-federation-plugin/src/api/Commands.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * This program and the accompanying materials are made available and may be used, at your option, under either:
+ * * Eclipse Public License v2.0, available at https://www.eclipse.org/legal/epl-v20.html, OR
+ * * Apache License, version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+import {IHandlerResponseApi, ImperativeError, TextUtils} from "@zowe/imperative";
+import { warn } from "console";
+import {IIdentity} from "./CsvParser";
+import {Constants} from "./Constants";
+import {hasValidLength} from "./ValidateUtil";
+
+export class Commands {
+
+    readonly maxLengthMainframeId = 8;
+    readonly maxLengthDistributedId = 246;
+    readonly maxLengthLabel = 32;
+
```

Comment:
Are these limits the same for all security packages?

Reply:
it is the same at least for TSS and RACF